<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>791</width>
      <height>591</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>model_load</string>
     </attribute>
     <widget class="QTextEdit" name="model_label">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>140</y>
        <width>291</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="model_load_btn">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>80</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>model_load</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>data_load</string>
     </attribute>
     <widget class="QPushButton" name="exit_btn">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>440</y>
        <width>60</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>실행</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="yLabel">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>140</y>
        <width>231</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="xLabel">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>140</y>
        <width>231</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ytest_btn">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>100</y>
        <width>230</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Y Test</string>
      </property>
     </widget>
     <widget class="QPushButton" name="xtest_btn">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>230</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>X Test</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
